### PR TITLE
Updates potato description to fit /vg/ culture

### PR DESCRIPTION
**Pull request by carshalash**
I can't let celt have all the memes to himself.

### CODE REVIEW

**Comment by RFultra** [comment]
Lmao yes

**Comment by ghost** [comment]
This is the second stupidest thing you've ever done

**Comment by carshalash** [comment]
I've done a lot of dumb things and this is not one of them.

**Comment by RFultra** [comment]
@DrCelt Care for some lemonade?

File: code/modules/reagents/reagent_containers/food/snacks/grown.dm [hunk 1]
```diff
@@ -329,7 +329,7 @@ var/list/special_fruits = list()
 
 /obj/item/weapon/reagent_containers/food/snacks/grown/potato
 	name = "potato"
-	desc = "Boil 'em! Mash 'em! Stick 'em in a stew!"
+	desc = "The space Irish starved to death after their potato crops died. Sadly they were unable to fish for space carp due to it being the queens space. Bringing this up to any space IRA member will drive them insane with anger."
```
Comment (Intigracy) [inline code/modules/reagents/reagent_containers/food/snacks/grown.dm]: queen's

**Review by Intigracy — CHANGES_REQUESTED** [review]

**Comment by PJB3005** [comment]
I want to merge this to spite Celt now.

On the other hand this is really dumb.

**Comment by ghost** [comment]
@PJB3005 Wouldn't be the first time you've done something exceptionally dumb just to spite me.

**Pull request closed without merging**